docs/source/how-tos/To-do-list-for-New-MOC-Intern-and-Member.md

```diff
@@ -1,62 +1,100 @@
  -  Subscribe to the [team mailing lists](https://lists.massopen.cloud)
- -  **Trello** -  : Send mail to msd@bu.edu with your Trello user name and you will be added to the appropriate organization.
- -  Once you are a member of the MOC team on Trello, go to the 
- [moc-massopencloud](https://trello.com/b/2IMwDpdi/moc-massopencloud) Trello board, and add a Trello card on the Backlog Trello list with
-     1. your full name
-     1. short paragraph about yourself
-     1. a photo of yourself
-
-See [example](https://trello.com/c/NF4xQCuq/102-sirine-benbrahim)
-
-### Team calendars
-We use Google calendars as a collaboration tool. We have the following calendars:
- -  **MOC** -  administrative
- -  **MOC - Activities** -  team-wide activities and meetings
- -  **MOC - Engineers** -  MOC engineers and embedded engineers' office hours, 
- OpenStack meetings that engineers attend on regular basis, etc.
- -  **MOC - Interns** -   interns/students' hours working as part of the MOC.
-
-Please share your the time slots when you plan to spend time in the MOC and/or on the MOC related projects.
-Send a request to msd@bu.edu to be added to the appropriate calendar.
+
+4. **Trello**: Send mail to msd@bu.edu with your Trello user name and you will be added to the appropriate organization.
+
+5. TODO: Contact Jen on how she would like to gather information on new people to publish to the website.
+
+### For Support Types
+
+Some of the items below require levels of access that must be earned over time, based on proving knowledge and judgement.  Leveling Up is encouraged!
+
+Please familiarize yourself with the following topics:
+
+1. OSTicket
+    1. Look at the knowledgebase to know what's already covered.
+2. Terminology
+    2. Kaizen
+    3. Openshift
+    4. Kubernetes
+    5. Engage1
+    6. Openstack
+    7. Version of SW
+        1. How to tell which Openstack
+        2. How to tell which openshift
+        3. How to tell
+    8. CEPH
+    9. Gluster
+    10. Object Store
+    11. Volume Store
+    12. Flavors
+    13. OVirt
+3. Openstack Tools
+    14. Horizon
+        4. Watch online tutorials, read up documentation. Would recommend creating instances, routers, security groups etc at least once.
+    15. Command Line
+        5. How to configure on your computer
+4. Monitoring System
+    16. Zabbix Dashboard navigation
+    17. Which zabbix URL for which information
+5. MAAS
+6. HIL/BMI
+7. FreeIPA
+8. Access to RH MOC account
+    18. How to research for existing bugs
+    19. How to open a ticket?  (Chris Wawak?)
+
+### For Software Developers
+
+1. (Start by reviewing pull requests - if you click approve you equally guilty as the person who wrote bad code that you approved)
+
+2. Suggestion: if writing python code, use some static analysis tools (e.g. pylint, black). Helpful in highlighting unused imports, variables, functions, bad return types etc before you submit your code for review.
+
+3. Read/understand the git workflow.
+
```

Comment:
Should mention:

4) Automated testing (unit/integration tests) to confirm that your project works
     - C++:  gtest 
     - python:  pytest

automated build/deploy - ideal, script out the entire build process so that there are no manual steps.  If there has to be a manual step, be sure to include it in both the documentation and in a comment at the top of the build script.

Reply:
I added this stuff, but I skipped adding C++ and gtest since we rarely use C++ for development.